modules/lib/src/main/resources/assets/js/app/util/SessionStorageHelper.ts

```diff
@@ -0,0 +1,66 @@
+import {ComponentPath} from '../page/region/ComponentPath';
+import {HtmlEditorCursorPosition} from '../inputtype/ui/text/HtmlEditor';
+
+export class SessionStorageHelper {
+    public static debug: boolean = false;
+
+    public static SELECTED_PATH_STORAGE_KEY: string = 'contentstudio:liveedit:selectedPath';
+
+    public static SELECTED_TEXT_CURSOR_POS_STORAGE_KEY: string = 'contentstudio:liveedit:textCursorPosition';
+
+    public static updateSelectedPathInStorage(contentId: string | undefined, value: ComponentPath | null): void {
```

Comment:
```suggestion
    public static updateSelectedPathInStorage(contentId: string | undefined, value?: ComponentPath): void {
```

Reply:
That is intentional, because **update** method needs an updated value. And you should not be able to update with no value.